packages/cli/src/domains/magento2/functions/checkNode.ts

```diff
@@ -0,0 +1,38 @@
+import { spawn } from 'child_process';
+import {
+  logSimpleErrorMessage,
+  logSimpleSuccessMessage
+} from './terminalHelpers';
+import { t } from 'i18next';
+
+const checkNodeVersion = (nodeString: string): boolean => {
```

Comment:
Not a fan of this solution, same as and forcing people to use Node.js version that will have [EoL in 5 months](https://endoflife.date/nodejs).

I'd say it would be much better to utilize built-it mechanisms, as described here https://stackoverflow.com/a/29349773/2859687

Reply:
Unfortunately, this is not my decision, out template runs on this version and we have to match it. It does not support version 17 or even 18. So, unless that is fixed, I can't do much